Source/WebInspectorUI/UserInterface/Views/InlineSwatch.js

```diff
@@ -382,11 +382,13 @@ WI.InlineSwatch = class InlineSwatch extends WI.Object
             codeMirror.setValue(value);
 
             const range = null;
+            const preventChangingColorFormats = this._preventChangingColorFormats;
```

Comment:
Style: We only use `const` for things that don't change between Web Inspector sessions (i.e. actual constants).  Please use `let`.